### ISSUE TITLE
HTML form control has no accessible name.

### ISSUE THREAD

**Issue by lfrancis** [usnationalarchives/nara-bia-photos#236]
HTML form control has no accessible name.

HTML form control has no accessible name.	WCAG 2.0 A F68 Section 508 (2017) A F68	1 pages
A label (or name) linked to the control allows screen readers to voice the label correctly when reading the control. To add a label do one of the following:

Use a label element with the for attribute set to the ID of the form control
Wrap a label element around the form control
Add a title attribute
Add an aria-label attribute
Add an aria-labelledby attribute

https://bia.test.drupalme.net/research/native-americans/bia/photos/states	Line 201

`<select class="sc-dvUxUk hAQeVd">`
